src/qml/components/BlockClock.qml

```diff
@@ -37,6 +37,18 @@ Item {
         backgroundColor: Theme.color.neutral2
         timeTickColor: Theme.color.neutral5
         confirmationColors: Theme.color.confirmationColors
+
+        Behavior on backgroundColor {
```

Comment:
@johnny9 should these be added to the cpp code, or do you think its fine here?

Reply:
It should be ok. It will create an edge case where some animations might update too quickly since I haven't implemented proper frame timing.

Reply:
I tried it out and I don't see a way to hit that edge case anyway.